packages/mobile/src/components/lineup/Lineup.tsx

```diff
@@ -140,8 +138,6 @@ export const Lineup = ({
     })
   }, disableTopTabScroll)
 
-  const playing = useSelector(getPlaying)
-  const playingUid = useSelector(getPlayingUid)
```

Comment:
so is the idea rather than making the whole lineup rerender, just the specific tile that actually needs to change color?

Reply:
Yup, rerendering the whole section list is v expensive. And the Lineup doesn't really need knowledge of which track is playing, only the individual tiles do